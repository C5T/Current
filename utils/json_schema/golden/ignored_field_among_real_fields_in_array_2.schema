// Autogenerated schema inferred from input JSON data.

CURRENT_STRUCT(Schema_Element_x_Object) {
  CURRENT_FIELD(a, std::string);
  // `b` : `null`-s and/or empty arrays, ignored in the schema.
  CURRENT_FIELD(c, std::string);
};

CURRENT_STRUCT(Schema_Element_Object) {
  CURRENT_FIELD(x, Optional<Schema_Element_x_Object>);
};

using Schema = std::vector<Schema_Element_Object>;
