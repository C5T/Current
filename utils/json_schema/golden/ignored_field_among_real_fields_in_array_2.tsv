Field	Type	Set	Unset/Null	Values	Details
Schema	Array	1	0
Schema[]	Object	4	0	1 field	x
Schema[].x	Object	3	1	3 fields	a, b, c
Schema[].x.a	String	3	0	1 distinct value, "foo":3
Schema[].x.b	Null	3
Schema[].x.c	String	3	0	1 distinct value, "bar":3
